src/components/Slider/Slider.tsx

```diff
@@ -311,7 +312,14 @@ const Slider: SliderComponentProps = ({
             onPressPoint={(point: number) => tapToSeek && onPressPoint(point)}
           />
         )}
-        <Tracked style={[trackedStyle, animatedTrackStyle]} />
+        <Track
+          style={StyleSheet.flatten([
+            styles.track,
```

Comment:
The `styles.track` is StyleSheet.absoluteFillObject so we do not need create this style